### DISCUSSION TITLE
Need help with IDH setup

### DISCUSSION THREAD

**Discussion by ammay0127** [Security-Onion-Solutions/securityonion#10703] [Unsupported Versions] [answered]
Hello everyone! Very new to Security Onion and currently trying to set up the IDH features. I've been following step by step with the youtube video below:

[HERE](https://www.youtube.com/watch?v=NzUhfARVfJk&list=PLljFlTO9rB17mESq7Z9OeFKvVh39vJW34&index=6).
(start timestamp around 16:30)

But by the time I get about a minute and a half into the process (the idh-setup.sh step), I get an error under -Available Forward Nodes- and I'm not sure where to go from there. I've attached a screencap of the terminal. Any advice would be appreciated!

![screen1](https://github.com/Security-Onion-Solutions/securityonion/assets/57539492/2e30a70b-3eff-4e69-86bc-c0703dba00ad)

**Comment by InfosecGoon** [comment]
That's actually a pre-release talk, from before the IDH functionality was folded into the platform as an official node type.

Do you have a functioning Security Onion deployment now? Either a Standalone, or a Manager with some minion nodes?

**Comment by ammay0127** [comment]
> That's actually a pre-release talk, from before the IDH functionality was folded into the platform as an official node type.
> 
> Do you have a functioning Security Onion deployment now? Either a Standalone, or a Manager with some minion nodes?

I currently have Security Onion deployed (version 2.3.240) with a few minion nodes
![nodes](https://github.com/Security-Onion-Solutions/securityonion/assets/57539492/34844f98-08d0-4be1-b746-3799d3f54aa5)

**Reply by InfosecGoon** [reply]
In that case, you should be able to just install a new node from the 2.3.240 ISO and designate it as an IDH node when you go through setup.

**Reply by ammay0127** [reply]
I don't quite understand what you mean by that... How can I go through setup again when Security Onion is already fully set up? Is there a command for that?

**Reply by InfosecGoon** [reply, marked as answer]
You need to install Security Onion on a separate node, either a physical computer or a VM, that will run the IDH services. When you boot the ISO, there should be an option in the GRUB menu to install an IDH node; use that option to install the Security Onion software. After the installation, when you reboot, it will go into setup and you can select "distributed", "add to an existing grid", and IDH node.
